pkg/util/encoding/BUILD.bazel

```diff
@@ -58,3 +59,19 @@ go_test(
         "@com_github_stretchr_testify//require",
     ],
 )
+
+# genrule(
```

Comment:
what happened here?

Reply:
It is showing this:
"stringer: can't handle non-integer constant type Type"

Reply:
You'll need to include `pkg/util/encoding/encodingtype` somehow, similar to what we do with x-package stringers here: https://github.com/cockroachdb/cockroach/blob/731d08354df5c04b6021cb9ba1f8ca901c69ff53/pkg/sql/opt/BUILD.bazel#L109

Reply:
Code is almost the same, I am using `$(location @org_golang_x_tools//cmd/stringer:stringer) -output=$@` inside `genrule` I might be confused there but what do you mean with "include `pkg/util/encoding/encodingtype` somehow"

The only difference I can see is that you are manually copy one of the source files over into the location of the other. So you mean something like copy `pkg/util/encoding/encodingtype` inside the same location in which we are running?

Reply:
the reason you are seeing this error is because `Type` is defined as

```
// Type represents the type of a value encoded by
// Encode{Null,NotNull,Varint,Uvarint,Float,Bytes}.
//go:generate stringer -type=Type
type Type encodingtype.T
```

which relies on

```
"github.com/cockroachdb/cockroach/pkg/util/encoding/encodingtype"
```

So yes, we'd need to copy the source files over as otherwise bazel's only source is `encoding.go` and has no way of reading `encodingtype`.